pkg/kubernetes/scripts/projects.js

```diff
@@ -978,25 +978,41 @@
         "dialogData",
         "projectData",
         '$location',
+        '$timeout',
         "kubeMethods",
-        function($q, $scope, kselect, dialogData, projectData, $location, methods) {
+        function($q, $scope, kselect, dialogData, projectData, $location, $timeout, methods) {
             var project = dialogData.project || { };
             var meta = project.metadata || { };
             var annotations = meta.annotations || { };
 
             var DISPLAY = "openshift.io/display-name";
             var DESCRIPTION = "openshift.io/description";
 
-            var shared;
-            if (meta.name)
-                shared = projectData.sharedImages(meta.name);
-
             var fields = {
                 name: meta.name || "",
                 display: annotations[DISPLAY] || "",
                 description: annotations[DESCRIPTION] || "",
-                access: shared,
+                access: null,
             };
+
+            /*
+             * This data may not be available yet, so continue to ask until it is.
+             * The called function takes care of not making duplicate requests.
+             */
+            var tim;
```

Comment:
Not wrong, but reading a name here threw me a bit - I didn't associate it with a timer thing initially.